app/views/DirectoryView/index.js

```diff
@@ -219,6 +219,7 @@ class DirectoryView extends React.Component {
 		}
 		return (
 			<DirectoryItem
+				title={item._id === 'GENERAL' ? item.name : item.fname}
```

Comment:
the item.fname can be undefined even if the _id is not 'GENERAL'. That make the title not showing up.

Reply:
@kresnaputra Hello ,yes I realised the mistake , and now I am trying to work around some other solution to make this work. But why is it so that in the main codebase server the item.name for discussion is is its name but for here it is the id